ChangeLog.d/systematically_store_bit_size_3740.txt

```diff
@@ -0,0 +1,7 @@
+Changes
+   * Always store key bits attribute in PSA key storage. Previously stored
+     keys should be augmented with the bits value to be compatible with the
+     updated version. #3740
+   * Removes the stored bits field from the storage format for the
```

Comment:
This is really a single change: store the key size systematically instead of just for dynamic secure element keys.